docs/configuring.md

```diff
@@ -0,0 +1,241 @@
+
+### z/OS systems  
+
+When available, 1 MB pageable pages are the default size for the object heap and the code cache. Other page sizes are available for the object heap, depending on the system architecture as shown in the following table:
+
+| Large page size  | System architecture required              | `-Xlp:codecache` | `-Xlp:objectheap`          |
+|------------------|-------------------------------------------|------------------|----------------------------|
+| 2 GB nonpageable | IBM zEnterprise EC12 processor or later   | Not supported    | Supported (64-bit VM only) |
+| 1 MB nonpageable | System z10 processor or later             | Not supported    | Supported (64-bit VM only) |
+| 1 MB pageable    | IBM zEnterprise EC12 processor or later (see Note) |Supported| Supported                  |
+
+<i class="fa fa-pencil-square-o" aria-hidden="true"></i> **Note:** The Flash Express feature (#0402) helps avoids demoting 1 MB pageable pages to 4 KB pages during periods of high memory usage.
+
+If a particular page size cannot be allocated, a smaller page size is attempted, in descending order. For example, if 2 GB nonpageable pages are requested but not available, the VM tries to allocate 1MB nonpageable pages. If 1 MB nonpageable pages are not available, the VM tries to allocate 1MB pageable pages. If large pages are not available, 4 KB pages are allocated.
+
+If you want to use nonpageable large pages for the object heap, a system programmer must configure z/OS for nonpageable large pages in the **IEASYxx** parmlib member. Users who require large pages must also be authorized to the **IARRSM.LRGPAGES** resource in the RACF FACILITY class with read authority.
+
+Use the following z/OS system command to show large page usage for an LPAR:
+
+```
+DISPLAY VS,LFAREA
+```
+For more information, see [LFAREA](https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.4.0/com.ibm.zos.v2r4.ieae200/lfareaxx.htm) in the z/OS product documentation.
+
+For usage information, including examples, see [`-Xlp:objectheap`](xlpobjectheap.md#pageablenonpageable).
+
+
+## Configuring Dynamic LPAR support (AIX only)
+
+*Dynamic logical partitioning* (DLPAR) provides a mechanism to add or remove system resources, such as memory or CPU, to or from the operating system in a logical partition without rebooting. Changing these resources dynamically can have an impact on Java applications that are running on the LPAR.
+
+To enable an application to respond to DLPAR events, you can use OpenJ9 extensions to the `java.lang.management` API. These extensions can listen for events and trigger any necessary adjustments to the runtime environment. For example, if a Java VM is running in an LPAR with 2GB of memory, but the available memory might be adjusted between 1GB and 8GB, you might set the following options for the Java heap at run time:
```

Comment:
This should refer to the actual extensions in the API docs.
AvailableProcessorsNotificationInfo
ProcessingCapacityNotificationInfo
TotalPhysicalMemoryNotificationInfo